polus/plugins.py

```diff
@@ -433,6 +442,11 @@ def cast_version(cls, value):
     def organization(self):
         return self.containerId.split("/")[0]
 
+    def save_manifest(self, path: typing.Union[str, pathlib.Path], indent: int = 4):
+        with open(path, "w") as fw:
+            json.dump(self.manifest, fw, indent=indent)
```

Comment:
Can we just do `self.dict()` instead of `self.manifest`?

If the issue is that we would also save field values, what we can do is set up excludes in the model definition to exclude model parameters. That way we can just have one "version" of a saving function: one version that saves the pure manifest, and another the saves the manifest but it embeds configuration values into it.

Also, if we just use `self.dict` for saving, it makes it super easy to reload the data into a model.